docs/usage/8.md

```diff
@@ -0,0 +1,298 @@
+
+```
+export noobaa_sha=$(grep NOOBAA_CORE_IMAGE -A1 manifests/ocs-operator/4.2.0/ocs-operator.v4.2.0.clusterserviceversion.yaml | grep value | awk -F "[ :]" '{ print $NF }')
+export ooperator_sha=$(grep ocs-rhel8-operator manifests/ocs-operator/4.2.0/ocs-operator.v4.2.0.clusterserviceversion.yaml | awk -F ':' '{ print $NF }')
+export roperator_sha=$(grep rook-ceph-rhel8-operator manifests/ocs-operator/4.2.0/ocs-operator.v4.2.0.clusterserviceversion.yaml | awk -F ':' '{ print $NF }')
+export moperator_sha=$(grep mcg-rhel8-operator manifests/ocs-operator/4.2.0/ocs-operator.v4.2.0.clusterserviceversion.yaml | awk -F ':' '{ print $NF }')
+export new_roperator_sha=$(oc image mirror --dry-run --insecure=true ${remote_registry}/${remote_registry_ns}/${roperator_image}@sha256:${roperator_sha} ${exposed_registry}/${local_registry_ns}/rook-ceph-rhel8-operator:v4.2.0 2>&1 | grep -A1 manifests | grep sha256 | awk -F'[: ]' '{ print $8 }')
+export new_ooperator_sha=$(oc image mirror --dry-run --insecure=true ${remote_registry}/${remote_registry_ns}/${ooperator_image}@sha256:${ooperator_sha} ${exposed_registry}/${local_registry_ns}/ocs-rhel8-operator:v4.2.0 2>&1 | grep -A1 manifests | grep sha256 | awk -F'[: ]' '{ print $8 }')
+export new_moperator_sha=$(oc image mirror --dry-run --insecure=true ${remote_registry}/${remote_registry_ns}/${moperator_image}@sha256:${moperator_sha} ${exposed_registry}/${local_registry_ns}/mcg-rhel8-operator:v4.2.0 2>&1 | grep -A1 manifests | grep sha256 | awk -F'[: ]' '{ print $8 }')
+export new_noobaa_sha=$(oc image mirror --dry-run --insecure=true ${remote_registry}/${remote_registry_ns}/${noobaa_image}@sha256:${noobaa_sha} ${exposed_registry}/${local_registry_ns}/mcg-core-rhel8:v4.2.0 2>&1 | grep -A1 manifests | grep sha256 | awk -F'[: ]' '{ print $8 }')
+sed -i "s/${roperator_sha}/${new_roperator_sha}/g" manifests/ocs-operator/4.2.0/ocs-operator.v4.2.0.clusterserviceversion.yaml
+sed -i "s/${moperator_sha}/${new_moperator_sha}/g" manifests/ocs-operator/4.2.0/ocs-operator.v4.2.0.clusterserviceversion.yaml
+sed -i "s/${ooperator_sha}/${new_ooperator_sha}/g" manifests/ocs-operator/4.2.0/ocs-operator.v4.2.0.clusterserviceversion.yaml
+sed -i "s/${noobaa_sha}/${new_noobaa_sha}/g" manifests/ocs-operator/4.2.0/ocs-operator.v4.2.0.clusterserviceversion.yaml
+```
+
+### Update the OCS operator source and namespaces for dev/stage
+host unknown errors seem to stop the image mirror policy from working? The same with namespace and permissions to it? These claims need verification.
+```
+if [[ "registry.redhat.io" != "${source_registry}" ]]; then
+  sed -i "s/${ocs_source_registry}/${source_registry}/g" manifests/ocs-operator/4.2.0/ocs-operator.v4.2.0.clusterserviceversion.yaml
+  sed -i "s/${ocs_source_registry_ns}/${source_registry_ns}/g" manifests/ocs-operator/4.2.0/ocs-operator.v4.2.0.clusterserviceversion.yaml
+fi
+```
+
+## Step 4-5: Skip
```

Comment:
Do you think it would be useful to explain why it's OK to skip these steps? I'll leave it to your judgement